topiary/args.py

```diff
@@ -132,50 +156,37 @@ def mhc_alleles_from_args(args):
 # MHC Binding Prediction
 #
 
-mhc_predictor_arg_group = arg_parser.add_mutually_exclusive_group(
+mhc_predictors = {
+    "netmhc": mhctools.NetMHC,
+    "netmhcpan": mhctools.NetMHCpan,
+    "netmhccons": mhctools.NetMHCcons,
+    "random": mhctools.RandomBindingPredictor,
+    # TODO implement SMM predictors in mhctools
+    "smm": None,
```

Comment:
To guilt myself into implementing them (https://github.com/hammerlab/mhctools/issues/7)